lib/zendesk_api/resources.rb

```diff
@@ -775,6 +775,12 @@ def self.singular_resource_name
 
   class Target < Resource; end
 
+  class Webhook < Resource; end
```

Comment:
Do we need this?  I thought this file is required for Classic API and we are going to fetch webhooks from Webhook API instead. I may misunderstand the context here. :)

Reply:
Interesting.

Without tests or any actual use of this code, I'm inclined to close this pull request.

Did you gain a better insight and do you think we should complete this now now? @token-cjg & @cl8au?

Reply:
Pegasus reached out about this recently, as it is desirable for this functionality to be within the ruby client so that they can action a requirement associated to a joint pending project. cc @zendesk/pegasus

I can look into writing one or two tests for this. Sorry got a bit addled and closed it out; I'll open this again and see if I can massage it into shape this afternoon.